lib/fastlyctl/commands/logging/bigquery.rb

```diff
@@ -0,0 +1,125 @@
+module BigQuery
+
+    def BigQuery.parse_secret_key(file)
+        key = File.read(file)
+        if key[0..26] != "-----BEGIN PRIVATE KEY-----"
+            abort "Error, private key file should begin with -----BEGIN PRIVATE KEY-----"
+        end
+
+        return key 
+    end
+
+    def BigQuery.ensure_opts(required_opts,options) 
+        required_opts.each { |k| 
+            if options[k].nil? 
```

Comment:
I try my best to avoid checking for nil, I just find it nonspecific. `options.key?(k)` works. Although im sure I've done exactly what you did many times 😂 

_this is not important and im just giving my thoughts on the style_